app/forms/idv/api_image_upload_form.rb

```diff
@@ -98,10 +98,12 @@ def post_images_to_client
     end
 
     def front_image_bytes
+      return unless front
```

Comment:
I get that we want this to be nil-safe.  What would you think about replacing the unless statement with`return if front.nil?`. I think that might be a little clearer.  (I saw Zach's comment about using a nil safe operator.  That works, too.)